### PR TITLE
Added default values

### PR DESCRIPTION
**Pull request by NathanSpeidel**

### CODE REVIEW

File: app/templates/partials/-output-data-help.hbs [hunk 1]
```diff
@@ -3,7 +3,7 @@
   Licensed under the terms of the Apache License, Version 2.0.
   See the LICENSE file associated with the project for terms.
  --}}
-<h5 class="content-header">Raw Fields</h5>
+<h5 class="content-header">Raw Data</h5>
```
Comment (akshaisarma) [inline app/templates/partials/-output-data-help.hbs:6]: Revert this line

File: app/templates/partials/-output-data-help.hbs [hunk 2]
```diff
@@ -63,8 +63,8 @@
...
+  errors are reported in the Result Metadata section in the Results Page. Please note that the values are <strong>exact</strong> up to a configured threshold, k, which is <strong>{{settings.defaultValues.countDistinctEntries}}</strong> by default. The value of Count Distinct is approximate after this threshold, with a <strong>bounded maximum error</strong>. For example, for k = 16,384, the error is at most 2.34% at 99.73% confidence. In other words, if your count distinct value was 2,000,000, the real value was somewhere between 1,953,200 and 2,046,800 at 99.73% confidence. These thresholds depend on what the maintainer has chosen for this instance of Bullet but these are the defaults. The error bounds for other values of k can be seen <a href="https://datasketches.github.io/docs/Theta/ThetaErrorTable.html">here</a>.
```
Comment (akshaisarma) [inline app/templates/partials/-output-data-help.hbs]: Remove this ```These thresholds depend on what the maintainer has chosen for this instance of Bullet but these are the defaults. ```

File: app/templates/partials/-output-data-help.hbs [hunk 3]
```diff
@@ -63,8 +63,8 @@
...
+  Another important feature to note is that Bullet has a maximum number of records it can return (also chosen by the maintainer). If you are choosing the Grouped Data option and your group fields have more unique combinations than this, you will get a <strong>uniform</strong> sample across these unique values. In other words, if there are more groups than can be returned, there is no guarantee of which of your group values and metrics (i.e. your result rows) will be returned. These should not affect the common queries that the maintainer expects people to perform on the data that Bullet is running on. You should reach out to the maintainer if you do find practical queries that cannot be performed adequately because of the limits the maintainer has chosen. These limits arise partly from the streaming nature of the data and computational intractability of keeping track of arbitrary numbers of unique values and partly from having to prevent users from overloading the system due to improperly created queries.
```
Comment (akshaisarma) [inline app/templates/partials/-output-data-help.hbs]: You should refer to the aggregateMaxSize setting here.

File: app/templates/partials/-result-metadata-help.hbs [hunk 4]
```diff
@@ -9,22 +9,22 @@
 
 <h5 class="content-header">Start Time</h5>
 <p>
-  This denotes the time Bullet received the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
+  The start time of the query is stored with the key <strong>{{settings.defaultValues.queryCreationTime}}</strong>. This denotes the time Bullet received the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
```
Comment (akshaisarma) [inline app/templates/partials/-result-metadata-help.hbs]: <strong>{{settings.defaultValues.queryCreationTime}}</strong> is the start time of the query.

File: app/templates/partials/-result-metadata-help.hbs [hunk 5]
```diff
@@ -9,22 +9,22 @@
...
 </p>
 
 <h5 class="content-header">Finish Time</h5>
 <p>
-  This denotes the time Bullet terminated the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
+  The finish time of the query is stored with the key <strong>{{settings.defaultValues.queryTerminationTime}}</strong>. This denotes the time Bullet terminated the query. This is useful to help you determine whether the data you were looking for had or had not reached Bullet yet.
```
Comment (akshaisarma) [inline app/templates/partials/-result-metadata-help.hbs]: Put the key name first as above

File: app/templates/partials/-result-metadata-help.hbs [hunk 6]
```diff
@@ -37,10 +37,10 @@
...
 </p>
 
 <h5 class="content-header">Sketch Theta</h5>
 <p>
-  This denotes the theta value of the Sketch that was used underneath to perform a Count Distinct or a Group operation. This is particularly useful if you have Grouped Data and you have more groups than can be returned. If you have performed metrics like Sum or Count, you can add them up over the returned rows in the Result below and divide by this Sketch Theta. The resulting number is an estimate of the Sum or Count across <strong>all</strong> your unique values that could not be returned.
-</p>
\ No newline at end of file
+  This value, stored with the key <strong>{{settings.defaultValues.theta}}</strong>, denotes the theta value of the Sketch that was used underneath to perform a Count Distinct or a Group operation. This is particularly useful if you have Grouped Data and you have more groups than can be returned. If you have performed metrics like Sum or Count, you can add them up over the returned rows in the Result below and divide by this Sketch Theta. The resulting number is an estimate of the Sum or Count across <strong>all</strong> your unique values that could not be returned.
```
Comment (akshaisarma) [inline app/templates/partials/-result-metadata-help.hbs]: Rephrase to having the key first.

File: config/env-settings.json [hunk 7]
```diff
@@ -12,7 +12,19 @@
       }
     ],
     "bugLink": "https://github.com/yahoo/bullet-ui/issues",
-    "aggregateDataDefaultSize": 512,
-    "modelVersion": 1
+    "modelVersion": 1,
+    "defaultValues": {
+      "aggregateMaxSize": 512,
+      "rawMaxSize": 30,
+      "maxDurationSec": 120,
```
Comment (akshaisarma) [inline config/env-settings.json]: maxDurationSecs

File: config/env-settings.json [hunk 8]
```diff
@@ -12,7 +12,19 @@
       }
     ],
     "bugLink": "https://github.com/yahoo/bullet-ui/issues",
-    "aggregateDataDefaultSize": 512,
-    "modelVersion": 1
+    "modelVersion": 1,
+    "defaultValues": {
+      "aggregateMaxSize": 512,
```
Comment (akshaisarma) [inline config/env-settings.json]: aggregationMaxSize

File: config/env-settings.json [hunk 9]
```diff
@@ -12,7 +12,19 @@
...
+      "rawMaxSize": 30,
+      "maxDurationSec": 120,
+      "countDistinctEntries": 16384,
+      "queryCreationTime": "query_receive_time",
```
Comment (akshaisarma) [inline config/env-settings.json]: These should be in a metadataKeyMapping map.

File: app/templates/partials/-options-help.hbs [hunk 10]
```diff
@@ -10,10 +10,10 @@
...
 <h5 class="content-header">Maximum Result Count</h5>
 <p>
-  The max number of results (records) returned for the Raw query by Bullet. By default, it needs to be a integer between <strong>1 and 30</strong>.
+  The max number of results (records) returned for the Raw query by Bullet. By default, it needs to be an integer between <strong>1 and {{settings.defaultValues.rawMaxSize}}</strong>.
 </p>
 
 <h5 class="content-header">Maximum Duration</h5>
 <p>
-  The max time Bullet will look for records matching the filter criteria. Must be a number between 1 and 120 seconds (default). Note that this is a not a guarantee to denote the results will arrive in this time, only that this is the window for gathering results. Typically, the results take a few seconds (~3 s) after the duration to be returned.
+  The max time Bullet will look for records matching the filter criteria. Must be a number between 1 and {{settings.defaultValues.maxDurationSec}} seconds (default). Note that this is a not a guarantee that the results will arrive in this time, only that this is the window for gathering results. Typically, the results take a few seconds (~3 s) after the duration to be returned.
```
Comment (akshaisarma) [inline app/templates/partials/-options-help.hbs]: The ~3 s part should be replaced with a reference to a new  joinBoltTickCount * tick duration setting.

File: app/templates/partials/-output-data-help.hbs (hunk 3 above)
Comment (akshaisarma) [reply app/templates/partials/-output-data-help.hbs]: Remove ```(also chosen by the maintainer)```. 

Remove```These should not affect the common queries that the maintainer expects people to perform on the data that Bullet is running on.```

**Review by akshaisarma — CHANGES_REQUESTED** [review]

File: app/templates/partials/-output-data-help.hbs (hunk 1 above)
Comment (NathanSpeidel) [reply app/templates/partials/-output-data-help.hbs:6]: You sure?  The field is called "Raw Data", I thought having the help say "Raw Fields" might be a little confusing...

Comment (akshaisarma) [reply app/templates/partials/-output-data-help.hbs:6]: Our PM wanted everything to be called Fields, I think. Let's keep this in if the section is called Raw Data and I'll reconcile them later.

File: app/templates/partials/-options-help.hbs (hunk 10 above)
Comment (NathanSpeidel) [reply app/templates/partials/-options-help.hbs]: Sounds good... right now the defaults: (tickIntervalSecs * tickTimeout) = (5 * 3) = 15 seems a little long.  Maybe we should also change the defualts in bullet-storm to maybe tickIntervalSecs = 1 or 2?  And I could change it in this PR as well?

**Comment by NathanSpeidel** [comment]
Review again?

**Review by akshaisarma — APPROVED** [review]

**Pull request merged**